src/Symfony/Component/Config/Definition/BooleanNode.php

```diff
@@ -20,10 +20,26 @@
  */
 class BooleanNode extends ScalarNode
 {
+    protected bool $nullable;
+
+    /**
+     * @param bool $nullable Whether the node value can be null
+     */
+    public function __construct(?string $name, ?NodeInterface $parent = null, string $pathSeparator = self::DEFAULT_PATH_SEPARATOR /* , bool $nullable = false */)
```

Comment:
constructors don't need the special trick to add new arguments in a BC way, as PHP does not enforce consistent signatures when overriding a constructor.

Reply:
Nice, let's update this part. It seems this case is missing in https://symfony.com/doc/current/contributing/code/bc.html#changing-classes? If yes, I'll send a PR in order to reflect that

Reply:
Indeed, we are missing the argument-related changes in the `Constructors` section.

Reply:
Done in 
* https://github.com/symfony/symfony-docs/pull/20307